### PR TITLE
Update n-body sources and binary for rust. Add bench results

### PR DESCRIPTION
**Pull request by MaxGraey**

### CODE REVIEW

File: examples/n-body/assembly/index.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ import "allocator/arena";
 
 type float = f64; // interchangeable f32/f64 for testing
 
-const SOLAR_MASS = <float>(4.0 * Math.PI * Math.PI);
+const SOLAR_MASS: float = 4.0 * Math.PI * Math.PI;
```
Comment (dcodeIO) [inline examples/n-body/assembly/index.ts]: I think this won't work if `type float = f32` because `Math.PI` is `f64`.

Comment (MaxGraey) [reply examples/n-body/assembly/index.ts]: Hmm, I thought `const a: type = b` and `const a = <type>(b)` has same result. Isn't true?

Comment (dcodeIO) [reply examples/n-body/assembly/index.ts]: Exceptions from this rule are `NaN` and `Infinity` that derive their type (either f32 or f64) from context atomatically.

Comment (MaxGraey) [reply examples/n-body/assembly/index.ts]: Got it. That's good to know

Comment (MaxGraey) [reply examples/n-body/assembly/index.ts]: Fixed!

**Comment by dcodeIO** [comment]
Thanks :)

**Pull request merged**